submit/submit.go

```diff
@@ -259,6 +259,7 @@ $BATCH_SCRIPT
 	if err != nil {
 		panic(errors.Wrap(err, "error registering job definition"))
 	}
+	defer deleteJobDefinition(b, ro)
```

Comment:
can you add a comment above this reminding us the we're discarding the error. doesn't matter most likely, but I want to remember that we're explicitly ignoring it for now.